src/helpers/helpers.canvas.js

```diff
@@ -64,88 +70,94 @@ var exports = module.exports = {
 			return;
 		}
 
-		ctx.save();
-		ctx.translate(x, y);
-		ctx.rotate(rotation * Math.PI / 180);
 		ctx.beginPath();
 
 		switch (style) {
 		// Default includes circle
 		default:
-			ctx.arc(0, 0, radius, 0, Math.PI * 2);
+			ctx.arc(x, y, radius, 0, DOUBLE_PI);
 			ctx.closePath();
 			break;
 		case 'triangle':
-			edgeLength = 3 * radius / Math.sqrt(3);
-			height = edgeLength * Math.sqrt(3) / 2;
-			ctx.moveTo(-edgeLength / 2, height / 3);
-			ctx.lineTo(edgeLength / 2, height / 3);
-			ctx.lineTo(0, -2 * height / 3);
+			ctx.moveTo(x + Math.sin(rad) * radius, y - Math.cos(rad) * radius);
+			rad += TWO_THIRDS_PI;
+			ctx.lineTo(x + Math.sin(rad) * radius, y - Math.cos(rad) * radius);
+			rad += TWO_THIRDS_PI;
+			ctx.lineTo(x + Math.sin(rad) * radius, y - Math.cos(rad) * radius);
 			ctx.closePath();
 			break;
-		case 'rect':
-			size = 1 / Math.SQRT2 * radius;
-			ctx.rect(-size, -size, 2 * size, 2 * size);
-			break;
 		case 'rectRounded':
-			var offset = radius / Math.SQRT2;
-			var leftX = -offset;
-			var topY = -offset;
-			var sideSize = Math.SQRT2 * radius;
-
-			// NOTE(SB) the rounded rect implementation changed to use `arcTo`
+			// NOTE(SB) the rounded rect implementation changed to use `arc`
```

Comment:
What's the difference between `arc` and `arcTo`?

Reply:
When I tried `arcTo`, there was no performance difference in Chrome and Safari, but it was slower than `arc` in FireFox. I guess there is an extra line rendering before the first `arcTo` call as `arcTo` requires a previous point in a subpath but `arc` doesn't. Also, the code is slightly simpler in case of `arc`.

Reply:
Just asking in case the drawing result was different but apparently not. I agree and I'm totally fine using `arc`, which also removes half function calls, so may be more efficient on some environments.